code-challenges.test.js

```diff
@@ -53,7 +78,23 @@ var vowelTester2 = "academy"
 // Expected output: 0
 var vowelTester3 = "challenges"
 // Expected output: 2
+//first im going to describe and im going to add my argument that is that i want  the index of the first vowel of my string.
+
+describe("vowelIndex", () =>{
+  //now im going to use it to describe what i want in return
+  it("return the index of the first vowel in a string",() =>{
+//now i do expect and here im going to write my function what i want but with especific data tomake my computer undestand me.
+    expect(vowelIndex(vowelTester1)).toEqual["1"]
+    expect(vowelIndex(vowelTester2)).toEqual["0"]
+    expect(vowelIndex(vowelTester3)).toEqual["2"]
+  })
+})
```

Comment:
I think you are super close. It looks like there is a syntax issue here `toEqual["1"]`
`.toEqual` is a method so you would need to use `()` and pass the same data type as the expected outcome.